src/components/v5/shared/MembersSelect/MembersSelect.tsx

```diff
@@ -35,6 +35,7 @@ const MembersSelect: FC<MemberSelectProps> = ({ user }) => {
       list={members}
       placeholderText={user ? undefined : { id: 'members.modal.selectMember' }}
       showAvatar
+      openButtonClass="outline-blue-100 outline !outline-[0.1875rem] border-blue-200"
```

Comment:
The same.

Also maybe we should create a special class in global for these styles because they are repeatable in a few components.